docs/use-the-network/devices/ready-to-use/rakwireless/rak7204.mdx

```diff
@@ -0,0 +1,164 @@
+---
+id: rak7204
+hide_title: true
```

Comment:
If you remove this line the title will show at the very top.

Reply:
So there's no need for me to actually change it now correct?

Reply:
Please remove the `hide_title: true` line or change true to false. This will allow the title to show up.

Reply:
Done 👍